src/Microsoft.Health.Dicom.Core/Features/Query/QueryLimit.cs

```diff
@@ -84,6 +90,10 @@ public static QueryTagLevel GetQueryTagLevel(DicomTag coreTag)
             {
                 return QueryTagLevel.Instance;
             }
+            if (CoreWorkitemTags.Contains(coreTag))
```

Comment:
Is it possible for extended query tags to overlap with the work item query tags? Eg. A user adds a work item query tag as an extended item query tag? Should we explicitly prevent this scenario in the extended query tag API?

Reply:
Yes it is possible for overlap. And we shouldn't restrict this scenario in ExtendedQueryTag since they index any standard tags they want. For workitem, we dont use level, when we refactor we need to change this.